README.md

```diff
@@ -23,18 +23,18 @@ In case neither of these sources are available, it will prompt the user via cons
 
 
 ## Required Python libraries
-Beyond the standard Python libraries, you need the 'requests' library.
+Beyond the standard Python libraries, you need the 'requests' and 'encryption' libraries.
 
-You can install it using pip:
+You can install them using pip:
 
 ```
-pip install requests 
+pip install requests encryption
```

Comment:
Does this package refer to [this one](https://pypi.org/project/Encryption/)? From the [code in the repo](https://github.com/nasiegel8/Encryptions/tree/master/Coders), it doesn't look like it

Reply:
That was a mistake; it was actually supposed to be `cryptography`. My bad.